components/brave_shields/resources/panel/theme/shields-dark.ts

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ import ITheme from 'brave-ui/theme/theme-interface'
+ import IThemeShields from './shields-theme'
+ import defaultDarkTheme from 'brave-ui/theme/brave-dark'
+
+ const shieldsDarkTheme: ITheme & IThemeShields = {
+   ...defaultDarkTheme,
+   name: 'Shields Dark',
+   color: {
+     ...defaultDarkTheme.color,
+     background03: '#3B3E4F',
+     text01: '#F0F2FF'
```

Comment:
Do these need to differ from the base theme? If the designer has labelled it as text01 then I'm confused why it would be different than the base, unless our base is still incorrect?

Reply:
Yes, our base has incorrect color for darkmode for text01.

Also, on Figma, the background in light mode points to: background02 but in dark mode its: background03

Reply:
That's weird - we should let design know when we see inconsistency like that. Aware we can't for this project, so ok on the separate theme override.